docs/src/guidelines.md

```diff
@@ -24,7 +25,8 @@ function guidelines_to_markdown_output(guidelines_function::Function)
     return output_markdown
 end
 
-const guidelines_function = RegistryCI.AutoMerge.get_automerge_guidelines_new_packages
+const guidelines_function = RegistryCI.AutoMerge.get_automerge_guidelines
+const registration_type = RegistryCI.AutoMerge.NewPackage()
```

Comment:
I didn't know you could define constants after they are used (I knew you could with functions) but seems like it works!

Reply:
FWIW, I don't think it's great for performance, but AutoMerge doesn't really have a lot of performance-sensitive code.

Reply:
The code isn't compiled until it's called and by that time the constants are defined (otherwise it would have been an error). It shouldn't affect the performance.